text/0037-ternary-numeric-conversion.md

```diff
@@ -0,0 +1,42 @@
++ Feature name: `ternary-numeric-conversion`
++ Start date: 2020-06-24
++ RFC PR: [iotaledger/bee-rfcs#37](https://github.com/iotaledger/bee-rfcs/pull/37)
++ Bee issue: [iotaledger/bee#93](https://github.com/iotaledger/bee/issues/93)
+
+# Summary
+
+Add a way to convert to and from binary types for ternary types.
+
+# Motivation
+
+Conversion between binary and ternary is often useful and so a good solution to this is required.
+
+# Detailed design
+
+This RFC introduces the following trait implementations:
+
+```rust
+// Binary to ternary
+impl<I: AsPrimitive<i64>, T: RawEncodingBuf> From<I> for TritBuf<T> {}
+
+// Ternary to binary (these cannot be implemented as above due to Rust's orphan rules)
+impl<'a, T: RawEncoding<Trit = Btrit> + ?Sized> TryFrom<&'a Trits<T>> for i64 {}
+impl<'a, T: RawEncoding<Trit = Btrit> + ?Sized> TryFrom<&'a Trits<T>> for i32 {}
+impl<'a, T: RawEncoding<Trit = Btrit> + ?Sized> TryFrom<&'a Trits<T>> for i16 {}
+impl<'a, T: RawEncoding<Trit = Btrit> + ?Sized> TryFrom<&'a Trits<T>> for i8 {}
+```
+
+The aforementioned trait implementations allow conversion to and from a variety of binary types for ternary types.
```

Comment:
Again, to avoid confusion I'd precise that we are speaking of numeric binary types

Reply:
Should we mention a solution regarding the `these cannot be implemented as above due to Rust's orphan rules`?